packages/web-app-files/src/views/spaces/DriveRedirect.vue

```diff
@@ -26,43 +21,21 @@ export default defineComponent({
       type: String,
       required: false,
       default: ''
-    },
-    appendHomeFolder: {
-      type: Boolean,
-      required: false,
-      default: false
     }
   },
   setup(props) {
     const store = useStore()
-    const userStore = useUserStore()
     const router = useRouter()
     const route = useRoute()
 
     const personalSpace = computed(() => {
       return store.getters['runtime/spaces/spaces'].find((space) => space.driveType === 'personal')
     })
 
-    const itemPath = computed(() => {
-      if (!props.appendHomeFolder) {
-        return ''
-      }
-      const item = props.driveAliasAndItem.startsWith(fakePersonalDriveAlias)
```

Comment:
This part is still needed. For reasoning see other comment on the variable declaration.